security_considerations.md

```diff
@@ -95,17 +95,16 @@ Many ECUs use EEPROM which, practically speaking, can be written to only a limit
 In order to analyze this problem, let us recap what new information should be downloaded in every software update cycle:
 
 1. The Primary writes and sends the latest vehicle version manifest to the Director repository.
-2. If a Time Server is used, all Secondaries write and send fresh tokens to the Primary.
-3. All ECUs download, verify, and write the latest downloaded time from the Time Server, or whatever source is used to provide the current accurate time.
-4. All ECUs download, verify, and write metadata from the Director and/or Image repositories.
-5. At some point, ECUs download, verify, and write images.
-6. At some point, ECUs install new images. Then, they sign, and write the latest ECU version reports.
+2. All ECUs download, verify, and write the latest downloaded time from the its source of current accurate time.
+3. All ECUs download, verify, and write metadata from the Director and/or Image repositories.
+4. At some point, ECUs download, verify, and write images.
+5. At some point, ECUs install new images. Then, they sign, and write the latest ECU version reports.
```

Comment:
These numbers can all be changed to `1.` and the markdown will figure out how to render the counting correctly. Then you don't have to change them all yourself every time. :)